sickbeard/search_queue.py

```diff
@@ -154,9 +158,11 @@ def run(self):
 
                     # give the CPU a break
                     time.sleep(common.cpu_presets[sickbeard.CPU_PRESET])
-
+            # What is this for ?!
             generic_queue.QueueItem.finish(self)
```

Comment:
I think i once added it to flag when A search was finished searching. But thats A long time ago.